salt/utils/aws.py

```diff
@@ -449,10 +450,11 @@ def query(params=None, setname=None, requesturl=None, location=None,
         )
         headers = {}
 
-    attempts = 5
-    while attempts > 0:
-        LOG.debug('AWS Request: {0}'.format(requesturl))
-        LOG.trace('AWS Request Parameters: {0}'.format(params_with_headers))
+    MAX_RETRIES = 6
+    attempts = 0
+    while attempts < MAX_RETRIES:
+        log.debug('AWS Request: %s', requesturl)
+        log.trace('AWS Request Parameters: %s', params_with_headers)
```

Comment:
Perhaps I'm overlooking something, but where is that lowercase `log` coming from?